CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to this project will be documented in this file. This project make usage of the [Yii Versioning Strategy](https://github.com/yiisoft/yii2/blob/master/docs/internals/versions.md).
 
+## 1.1.0
```

Comment:
@JohnnyMcWeed Why is this a major release change? Its not backwards compatible? The changes looks backwards compatible to me - except of the fact it can have more config options.

Reply:
I wasn't sure...
`$image['image']->caption` is now `$image['caption']` like the others